gamestonk_terminal/portfolio/portfolio_analysis/pa_controller.py

```diff
@@ -2,14 +2,26 @@
 
 import argparse
 import os
+from pathlib import Path
+
 import pandas as pd
 from prompt_toolkit.completion import NestedCompleter
 
 from gamestonk_terminal import feature_flags as gtff
-from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.helper_funcs import (
+    get_flair,
+    parse_known_args_and_warn,
+)
 from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio.portfolio_analysis import (
+    portfolio_model,
+    portfolio_view,
+)
 
-from gamestonk_terminal.portfolio.portfolio_analysis import portfolio_parser
+portfolios_path = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)), "portfolios/"
+)
+possible_paths = [Path(port).stem for port in os.listdir(portfolios_path)]
```

Comment:
You should check here that the files end with .csv since you add that after

Reply:
So this possible_ paths is just the choices for load.  When I do `view` I allow the user to specify file format (ie csv or json etc) which is then `if endsith(xyz)`- as one of the next steps is to allow from loading other files, not just csv.